backend/entrypoint

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-make -j 4
+make -j
```

Comment:
why do we reduce the number of parallel jobs? Will this not delay the make process?

Reply:
not specifying it means it uses all possible cores